hcl/hcl.go

```diff
@@ -102,7 +102,6 @@ const (
 	ErrHCLSyntax                errutil.Error = "HCL syntax error"
 	ErrMalformedTerramateConfig errutil.Error = "malformed terramate config"
 	ErrMalformedTerraform       errutil.Error = "malformed terraform"
-	ErrStackInvalidRunOrder     errutil.Error = "invalid stack execution order definition"
```

Comment:
All other specific configs don't have specific sentinels, so for now I kept everything consistently under ErrMalformedTerramateConfig, but we can review this later, if we really need more specific sentinels etc.